napari/_qt/dialogs/preferences_dialog.py

```diff
@@ -190,14 +192,23 @@ def get_page_dict(self, field: 'ModelField'):
 
     def restore_defaults(self):
         """Launches dialog to confirm restore settings choice."""
-        self._reset_dialog = ConfirmDialog(
-            parent=self,
-            text=trans._("Are you sure you want to restore default settings?"),
+
+        restore_box = QMessageBox()
+
+        settings = get_settings()
+        restore_box.setStyleSheet(get_stylesheet(settings.appearance.theme))
```

Comment:
is it still necessary to explicitly set the stylesheet if you give a parent to the QMessageBox?

```py
response = QMessageBox.question(
    self,
    trans._("Restore Settings"),
    trans._("Are you sure you want to restore default settings?"),
    QMessageBox.RestoreDefaults | QMessageBox.Cancel,
    QMessageBox.RestoreDefaults
)

if response == QMessageBox.RestoreDefaults:
    self._reset_widgets()
```

Reply:
@tlambert03 You're right, you don't need to do that.  setting the parent works great.  

Another question---If you use the QMessageBox.question function, you will get a symbol on the dialog--like this:
<img width="607" alt="Screen Shot 2021-08-03 at 10 05 10 AM" src="https://user-images.githubusercontent.com/54282105/128039578-855e7008-6a3d-4015-951a-cdf2b33f888a.png">

Using a plain QMessageBox, you get no symbol:
<img width="485" alt="Screen Shot 2021-08-03 at 10 06 44 AM" src="https://user-images.githubusercontent.com/54282105/128039658-d98b0d24-e472-450f-bc8a-2f2ab3a4e2f1.png">

which do you like better?

Reply:
I like the question (that's why used the static .question method), but don't have a strong preference, you can remove the icon if others prefer to

Reply:
I will leave it with the question for now.  The code is more concise that way.  :)